src/contexts/contracts/contracts-provider.tsx

```diff
@@ -1,10 +1,12 @@
 import {
+  Networks,
   VegaClaim,
   VegaErc20Bridge,
   VegaStaking,
   VegaToken,
   VegaVesting,
 } from "@vegaprotocol/smart-contracts-sdk";
+import { ethers } from "@vegaprotocol/smart-contracts-sdk/node_modules/ethers";
```

Comment:
Hmmm not a massive fan of this, can we make this a peer dependency from the smart contracts SDK?

Reply:
Or externalize, whichever is more appropriate

Reply:
Think this was an accidental auto import. Its gone now